docs/README.md

```diff
@@ -1,58 +1,82 @@
-# Scylla documentation
+# ScyllaDB Documentation
 
-This is the repository for [Scylla documentation site](http://docs.scylladb.com/)
+This repository contains the source files for ScyllaDB Open Source documentation.
 
-To report an issue with the documentation, please use GitHub issues.
+- The `dev` folder contains developer-oriented documentation related to the ScyllaDB code base. It is not published and is only available via GitHub.
+- All other folders and files contain user-oriented documentation related to ScyllaDB Open Source and are sources for [docs.scylladb.com](https://docs.scylladb.com/).
 
-This repository accepts GitHub pull requests.
+To report a documentation bug or suggest an improvement, open an issue in [GitHub issues](https://github.com/scylladb/scylla/issues) for this project.
```

Comment:
better point to the in page "report an issue" button

Reply:
Why? Why have a link to a button instead of the target of that button?

Reply:
@tzach @nyh I'd rather leave the link to the target as is. I wanted to explain how to work in this repo. 
Using the button is convenient, but the user doesn't need to care where the issue gets opened, so I feel we shouldn't focus on it in this very case.